examples/clients/abe/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+install:
+  - go get -d -v .
+
+script:
+  - go build -v ./
+
```

Comment:
This file shouldn't have been generated :thinking:. Could you delete it please?

Reply:
ok